ArcGISRuntimeSDKQt_CppSamples/Analysis/DisplayUtilityAssociations/DisplayUtilityAssociations.h

```diff
@@ -0,0 +1,80 @@
+// [WriteFile Name=DisplayUtilityAssociations, Category=Analysis]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef DISPLAYUTILITYASSOCIATIONS_H
+#define DISPLAYUTILITYASSOCIATIONS_H
+
+#include "UtilityNetworkTypes.h"
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+class GraphicsOverlay;
+class Map;
+class MapQuickView;
+class Symbol;
+class UtilityNetwork;
+}
+}
+
+class SymbolImageProvider;
+
+#include <QObject>
+
+class DisplayUtilityAssociations : public QObject
+{
+  Q_OBJECT
+
+  Q_PROPERTY(Esri::ArcGISRuntime::MapQuickView* mapView READ mapView WRITE setMapView NOTIFY mapViewChanged)
+  Q_PROPERTY(QString attachmentSymbolUrl READ attachmentSymbolUrl NOTIFY attachmentSymbolUrlChanged)
+  Q_PROPERTY(QString connectivitySymbolUrl READ connectivitySymbolUrl NOTIFY connectivitySymbolUrlChanged)
+  Q_PROPERTY(bool swatchesCompleted READ swatchesCompleted NOTIFY swatchesCompletedChanged)
```

Comment:
as mentioned above - maybe you don't need this property since you have a url for each swatch which will become non-empty